### ISSUE TITLE
508-defect-4: FormSignature component should have an axe check in unit test file

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#26266]
# [508-defect-4](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-4)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Details

I'd like each component to include an `axeCheck()` in the unit test spec. This will help us ensure continued compliance and warn of potential breaking changes. 

## Acceptance Criteria

- [ ] `axeCheck()` call added to test spec file
- [ ] 0 axe violations thrown

## Proposed Solution (if known)

```html
it('should pass aXe check', () => 
  axeCheck(<FormSignature />)
);
```

**Comment by taylorkaren** [comment]
Info from @micahchiang: Forms Library Team could add axe checks to the tests but this isn’t urgent [assumption is that we would add to tests for the FormSignature component only].
